src/store/rad.js

```diff
@@ -63,15 +62,14 @@ export default {
     clearSubscriptIds: function(state) {
       state.subscriptIds = []
     },
-    setDataRequestChangedSinceTried(state, payload) {
-      state.dataRequestChangedSinceTried = payload.value
-    },
     setDataRequestChangedSinceSaved(state, payload) {
       state.dataRequestChangedSinceSaved = payload.value
     },
     toggleTryDataRequest(state) {
       state.autoTry = !state.autoTry
-      state.dataRequestChangedSinceTried = true
+      state.autoTry
+        ? this.dispatch('tryDataRequest')
+        : this.commit('clearDataRequestResult')
```

Comment:
IMO toggle "auto try data request" shouldn't  remove  the last data request result

```suggestion
if (state.autoTry) this.dispatch('tryDataRequest')
```